### DISCUSSION TITLE
Add Multisite pages

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7057] [Troubleshooting] [answered]
I'm creating a multisite and have 3 separate sites. However I just want to create a single page in one of the sites, however it keeps copying and cloning and linking the page in the other sites also. So I have to keep going back into the sites to delete or remove the page or reference.  How can I turn this feature off where if I just create a single page in 1 site it just adds the single page in that site?

**Comment by joshuablum** [comment, marked as answer]
Hey @ottz0,

I think what you're looking for is the option to disable "propagation" for the collection. That essentially means that if you create an entry it gets automatically created for all of the collection's configured sites.

You can access it either via the Control Panel by clicking "Configure Collection" or edit the collection's YAML file directly.
<img width="989" alt="CleanShot 2022-11-15 at 1 17 31@2x" src="https://user-images.githubusercontent.com/3824203/201918262-50545281-c6b9-4c8b-92f2-dd469e39edf9.png">

```yaml
propagate: false
```

Hope that helps :)
